site/styles/_carousel.scss

```diff
@@ -63,10 +63,6 @@ s72-carousel {
       }
     }
   }
-
-  .s72-pricing-button-container {
-    white-space: nowrap;
```

Comment:
This might be the removed style that is causing the button wrapping that @PeculiarGoat noticed.

Reply:
Linter moved my css that wasn't inside the block disabling max compound selectors